### ISSUE TITLE
Cannot find module 'path/posix'

### ISSUE THREAD

**Issue by lpassos** [ConsenSys/scribble#175]
After installing scribble (version 0.6.9) and trying to run it, I got the following error: `Error: Cannot find module 'path/posix'`. However, I do have `path/posix` installed as a global package in my nodejs setup.

After inspecting `eth-scribble/dist/bin/scribble.js`, I noticed that the require statement for that module is incorrect. It currently states:

```
const posix_1 = require("path/posix");
```

whereas it should be

```
const posix_1 = require("path-posix");
```

After manually patching the require statement as indicated above, scribble worked as expected.

**Comment by blitz-1306** [comment]
Thanks for reporting. Will investigate. Seems to be an easy fix, but lets dig into it first.

**Comment by blitz-1306** [comment]
I've created PR to fix the issue: #177

**Comment by blitz-1306** [comment]
Please, check new version of Scribble: https://github.com/ConsenSys/scribble/releases/tag/v0.6.10. The bug should be fixed. Thanks for the collaboration.
